org.adempiere.base/src/org/compiere/model/MProcessPara.java

```diff
@@ -351,7 +350,13 @@ && getAD_Element_ID() != 0 && isCentrallyMaintained())
 				LogicEvaluator.validate(getDisplayLogic());
 			}
 		}
-		
+
+		if (newRecord && DisplayType.isChosenMultipleSelection(getAD_Reference_ID())) {
+			MProcess p = MProcess.get(getAD_Process_ID());
+			if (Util.isEmpty(p.getClassname()) && Util.isEmpty(p.getJasperReport()))

```

Comment:
Here you must check also that ProcedureName is empty

Not sure about the impact on Workflow and Special Form - must those be empty too?